endpoints/tests/test_music.py

```diff
@@ -109,6 +113,37 @@ def test_post(self):
 
         Music.objects.filter(music_id='a', room=self.r).exists().should.be.true
 
+    def test_post_existing_music(self):
+        m = Music(
+            music_id="a",
+            name="a",
+            thumbnail="https://a.com",
+            total_duration=114,
+            duration=114,
+            url="https://www.a.com",
+            source="youtube",
+            room=self.r,
+        )
+        m.save()
+
+        Music.objects.filter(music_id='a', room=self.r).first().duration.should.be.eql(114)
```

Comment:
This assertion is useless:/
You're testing the .save() method that is already tested by django itself